jbake/templates/index.ftl

```diff
@@ -110,70 +110,64 @@
         </a>
       </div>
 
-      <div class="panel panel-primary">
-        <div class=" panel-heading" style="text-align: center;">
-          <h3>Latest news:</h3>
+      <div class="card border-primary mb-4 shadow-sm">
+        <div class="card-header bg-primary text-white" style="text-align: center;">
+          <h3 class="card-title">Latest news:</h3>
         </div>
 
-        <div class="panel-body">
+        <div class="card-body">
         <#list posts as post>
         <#if (post.status == "published"  && post?index >= (currentPageNumber-1) * config.index_posts_per_page?eval && post?index < currentPageNumber * config.index_posts_per_page?eval)>
           <div>
-            <a href="${post.uri}"><small><h4>${post.title}</h4></small></a>
+            <a href="${post.uri}"><small>${post.title}</small></a>
             <p><small>by ${(post.author)!"The Apache Shiro Team"} on ${post.date?string("yyyy-MM-dd")}</small></p>
           </div>
         </#if>
         </#list>
           <hr/>
-          <nav aria-label="News Pagination">
-            <ul class="pagination">
+          <div class="btn-group d-flex justify-content-center" role="group" aria-label="News Pagination">
               <#if (currentPageNumber > 1)>
-              <li class="page-item"><a class="page-link" href="<#if (content.rootpath)??>${content.rootpath}</#if>/${(currentPageNumber==2)?then('', currentPageNumber-1)}">Previous</a></li>
+                <a class="btn btn-outline-primary" href="<#if (content.rootpath)??>${content.rootpath}</#if>/${(currentPageNumber==2)?then('', currentPageNumber-1)}">Previous</a>
               </#if>
               <#if (currentPageNumber > 1 && currentPageNumber != numberOfPages)>
-              <li class="page-item"><a class="page-link" href="${currentPageNumber}">${currentPageNumber}</a></li>
+                <a class="btn btn-outline-primary " href="${currentPageNumber}">${currentPageNumber}</a>
               </#if>
               <#if (currentPageNumber < numberOfPages)>
-              <li class="page-item"><a class="page-link" href="<#if (content.rootpath)??>${content.rootpath}</#if>${currentPageNumber + 1}">Next</a></li>
+                <a class="btn btn-outline-primary" href="<#if (content.rootpath)??>${content.rootpath}</#if>${currentPageNumber + 1}">Next</a>
               </#if>
```

Comment:
Oh yeah that also works. Anything wrong with following https://getbootstrap.com/docs/5.0/components/pagination/ though? Did it not work for you? (style-wise)

Reply:
I need to fix those links btw :(

Reply:
Oh nice! I didn't see that component! I'll tweak it!

Reply:
Actually, I think I was just trying to fix the centering and didn't realize it was an existing component 🤦